data/sdsswork.cfg

```diff
@@ -271,69 +271,70 @@ sdR_N = $BOSS_SPECTRO_DATA_N/{mjd}/sdR-{br}{id}-{frame}.fit
 sdR_S = $BOSS_SPECTRO_DATA_S/{mjd}/sdR-{br}{id}-{frame}.fit
 
 #boss DRP paths
-spplan2d = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spplan2d-@pad_fieldid|-{mjd}.par
-spPlancomb = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spPlancomb-@pad_fieldid|-{mjd}.par
+spplan2d = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/spPlan2d-@pad_fieldid|-{mjd}.par
+spPlancomb = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/spPlancomb-@pad_fieldid|-{mjd}.par
 conflist = $BOSS_SPECTRO_REDUX/{run2d}/conflist.fits
-fieldlist = $BOSS_SPECTRO_REDUX/{run2d}/fieldlist-{run2d}.fits
-spAll = $BOSS_SPECTRO_REDUX/{run2d}/spAll-{run2d}.fits
-spAll-lite = $BOSS_SPECTRO_REDUX/{run2d}/spAll-lite-{run2d}.fits
-spAllLine = $BOSS_SPECTRO_REDUX/{run2d}/spAllLine-{run2d}.fits
-spArc = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spArc-{br}{id}-{frame:0>8}.fits.gz
-spCFrame = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spCFrame-{br}{id}-{frame:0>8}.fits.gz
-spFlat = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spFlat-{br}{id}-{frame:0>8}.fits.gz
-spFluxcalib = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spFluxcalib-{br}{id}-{frame:0>8}.fits.gz
-spFluxcorr = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spFluxcorr-{br}{id}-{frame:0>8}.fits.gz
-spFluxdistort = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spFluxdistort-@pad_fieldid|-{mjd}-{frame:0>8}.fits
-spFrame = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spFrame-{br}{id}-{frame:0>8}.fits.gz
+fieldlist = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/fieldlist-{run2d}.fits
+spAll = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/spAll-{run2d}.fits
+spAll-lite = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/spAll-lite-{run2d}.fits
+spAllLine = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/spAllLine-{run2d}.fits
+spArc = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/spArc-{br}{id}-{frame:0>8}.fits.gz
+spCFrame = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/spCFrame-{br}{id}-{frame:0>8}.fits.gz
```

Comment:
I'm guessing for backwards compatibility, `sptypefolder` and `fieldgrp` functions will return empty strings.

Reply:
yes, that is correct.